users/spidey3/spidey3.c

```diff
@@ -192,6 +192,72 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
                 spi_gflock = !spi_gflock;
                 dprintf("spi_gflock = %u\n", spi_gflock);
                 break;
+
+            case KC_PSCR: {
+                uint8_t mods = get_mods();
+#ifndef NO_ACTION_ONESHOT
+                uint8_t osm = get_oneshot_mods();
+#else
+                uint8_t osm = 0;
+#endif
+
+                switch (get_unicode_input_mode()) {
+                    case UC_MAC:
+                        if ((mods | osm) & MOD_MASK_ALT) {
```

Comment:
If you want to check for the exact mods, you want:
```suggestion
                        if (((mods | osm) & MOD_MASK_ALT) == MOD_MASK_ALT) {
```

Reply:
I don't think that's right.
`(((mods | osm) & MOD_MASK_ALT) == MOD_MASK_ALT)` would require both KC_LALT and KC_RALT to be on.
`((mods | osm) & MOD_MASK_ALT)` checks for either KC_LALT or KC_RALT.

Reply:
Note similar code at https://github.com/qmk/qmk_firmware/blob/master/quantum/process_keycode/process_grave_esc.c#L31